### PR TITLE
Patch comments

### LINKED ISSUE #35: TODO, etc, not detected in single-line comments
Here is the issue:

in 

~~~ julia
# TODO do a thing
~~~

The `TODO`  should be highlighted. It it not. By comparison with

~~~ julia
#=
TODO do a thing
=#
~~~

where it is. `TODO` and a few others are part of `language-todo`, which is standard in atom.

### PR DESCRIPTION
**Pull request by tpoisot**
Fixes #35 -- and adds some specs, too

### CODE REVIEW

**Comment by tpoisot** [comment]
the `travis.yml` file may be broken

**Comment by spencerlyon2** [comment]
Thanks!

Looks like your last spec failed on Travis. Can you diagnose why?

**Comment by tpoisot** [comment]
Mmm. It seems that the specs are not correct. Let me try again.

**Comment by spencerlyon2** [comment]
Thanks, sorry I forgot to merge this earlier. I didn't see that you fixed the specs
